modules/vulnerability-scores.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images.adoc
+:_module-type: CONCEPT
+[id="vulnerability-scores_{context}"]
+= Vulnerability scores
+
+[role="_abstract"]
+In the RHACS portal, {product-title} shows a single CVSS base score for each vulnerability.
+{product-title} shows the CVSS score based on the following criteria:
+
+* If a CVSS v3 score is available, {product-title} shows the score and lists `v3` along with it.
+For example, `6.5 (v3)`.
+* If a CVSS v3 score is not available, {product-title} shows only the CVSS v2 score.
+For example, `6.5`.
+* You can use the API to get the CVSS scores.
+If CVSS v3 information is available for a particular CVE, the response includes both CVSS v3 and CVSS v2 information.
```

Comment:
This is the first reference to "CVE" in this module. You could state "Common Vulnerabilities and Exposures (CVE)" for the first reference, in case the module is reused in another assembly in the future which does not previously reference the full description.